proseco/characteristics_guide.py

```diff
@@ -109,7 +109,7 @@
           {"Stage": 6,
            "SigErrMultiplier": 0,
            "ASPQ1Lim": 20,
-           "MagLimit": [5.6, 10.3],
+           "MagLimit": [5.2, 10.3],
```

Comment:
For this one entry, I've only shifted it brighter by 0.4 mags instead of the 0.6 that I used for the rest.  But for the guide code, I think the candidate cut was previously at 5.8 anyway, so I don't know if the 5.6 here (brighter than that) in the stages was actually doing anything.  Will check.  The question is really just "should the new value here be 5.0? and should that be supported in the candidate selection code".

Reply:
In testing and thorough reading of the code, it is more clear that when doing the "bright" limit checks, the effective SigErrMultiplier is always 1 and the star mag error is handled as at least 0.1 mags.  Thus the MagLimit here can be a bit brighter than 5.2 to allow stars in later states as bright *as* 5.2 .